### PR TITLE
Jasper Documentation V1

### PR DESCRIPTION
**Pull request by Lama1403**
An update to the new content structure for documentation.

Irrelevant pages have not been deleted yet, just removed from main menu.

Some minor UI fixes done as well.

### CODE REVIEW

File: zq2/docs/endpoints.md [hunk 1]
```diff
@@ -1,29 +1,41 @@
...
 
-Devnet is a general purpose developer network, synchronised to the latest (or nearly the latest) commit from [https://github.com/zilliqa/zq2](https://github.com/zilliqa/zq2) . Its state and uptime are not guaranteed, but it is useful for checking your contracts against the latest and greatest software versions.
+## Block Explorer
 
-<div class="center-table" markdown>
-|  Name  | Version | ChainId | API | Otterscan | Faucet  | Connect | Notes |
-| ------ | ------- | ------- | --- | --------- | ------  | ------- | ----- |
-| devnet | <span id="devnet_vsn" class="zq2_vsn">api.zq2-devnet.zilliqa.com</span> | [33469](https://chainlist.org/chain/33469) | https://api.zq2-devnet.zilliqa.com | https://explorer.zq2-devnet.zilliqa.com | https://faucet.zq2-devnet.zilliqa.com/ | <a href="javascript:connectZilliqaChain('https://api.zq2-devnet.zilliqa.com', '0x82BD', 'Zilliqa 2 EVM Devnet', 'https://explorer.zq2-devnet.zilliqa.com', 'Zilliqa 2 testnet', 'ZIL')">Metamask</a> | <span class="zq2_docs_devnet_vsn" kind="api">&nbsp;</span> |
```
Comment (rrw-zilliqa) [inline zq2/docs/endpoints.md:26]: Could we please keep, if not this table, then the <span's> ? They provide the links developers need to interact with the testnet, and automatically query the version currently running.

File: zq2/docs/sdk.md [hunk 2]
```diff
@@ -1,53 +1,44 @@
...
-- [zilliqa-developer](https://github.com/Zilliqa/zilliqa-developer) - this contains source for, among other things
-  - [eth-spout](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/eth-spout) a simple faucet.
-  - [xbridge](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/bridge) a cross-chain contract bridge.
-  - [multisig](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/multisig) - a scilla multisig implementation
-  - [neo-savant](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/neo-savant) - the Scilla IDE
-  - [pdt](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/pdt) - a utility which turns the state of the blockchain into a BigQuery dataset or PostgreSQL database.
-  - [developer-portal-zq2](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/developer-portal-zq2) - this developer portal
+- [eth-spout](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/eth-spout) - a simple faucet.
+- [neo-savant](https://github.com/Zilliqa/zilliqa-developer/tree/main/products/neo-savant) - the Scilla IDE
+- [developer-portal-zq2](#) - this developer portal
```
Comment (rrw-zilliqa) [inline zq2/docs/sdk.md:44]: I think: 
- the developer portal link ought to be to this git repo (admittedly not the link inthe docs!)
- what did multisig do wrong? (it should work as long as scilla does)
-

**Review by rrw-zilliqa — DISMISSED** [review]
Couple of comments, which I can fix in a future PR

File: zq2/docs/sdk.md (hunk 2 above)
Comment (rrw-zilliqa) [reply zq2/docs/sdk.md:44]: https://github.com/Zilliqa/dev-portal/issues/23

File: zq2/docs/endpoints.md (hunk 1 above)
Comment (rrw-zilliqa) [reply zq2/docs/endpoints.md:26]: https://github.com/Zilliqa/dev-portal/issues/23

**Review by frankmeds — APPROVED** [review]

**Pull request merged**